src/server/frontend_wayland/wl_pointer.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "wayland_utils.h"
 #include "wl_surface.h"
+#include "wl_seat.h"
```

Comment:
This is just to access WlSeat::timestamp_of()?

Reply:
Yes, but it will get included by the next PR regardless (see other comment)